src/calculateRentalCost.js

```diff
@@ -29,6 +29,17 @@
 
 function calculateRentalCost(days) {
   // write code here
+  if (days < 3) {
+    return days * 40;
+  }
+
+  if (days < 7) {
+    return days * 40 - 20;
+  }
+
+  if (days >= 7) {
+    return days * 40 - 50;
+  }
 }
 
```

Comment:
```
 if (days < 3) {
    return days * 40;
  } else if (days < 7) {
    return days * 40 - 20;
  }
  return days * 40 - 50;
```

Reply:
@Bogdan-Mykhailov why do we need to use `else if` instead of just `if`?

Reply:
@Andr1yK if-else statements stop doing comparisons as soon as it finds one that's true.
if-if does every comparison.

Reply:
> if-if does every comparison

Yes, but not after return :)